pkg/volume/iscsi/iscsi.go

```diff
@@ -97,6 +97,11 @@ func (plugin *iscsiPlugin) newBuilderInternal(spec *volume.Spec, podUID types.UI
 	lun := strconv.Itoa(iscsi.Lun)
 	portal := portalBuilder(iscsi.TargetPortal)
 
+	iface := iscsi.ISCSIInterface
+	if iface == "" {
+		iface = "default"
```

Comment:
Each API version will have a defaults.go file where you can add this.

https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/defaults.go#L148